### DISCUSSION TITLE
Setting a static IP for the create 3

### DISCUSSION THREAD

**Discussion by hfhelmich** [iRobotEducation/create3_docs#218] [Questions]
I'm trying to get a robot on my department's default network and then set a static IP for that one robot. I can do this on other networks where I have admin capabilities, but I don't have those capabilities on this network I'd like to connect to. One option is to SSH into the robot on my desired network, but I'm prompted a username & password on the robot that I can't get around. Is it possible to retrieve the username & password somewhere? If there are other workarounds to this problem that were successful, please let me know! Thanks!

**Comment by shamlian** [comment]
The Create 3 robot gets its wlan0 IP address via DHCP. It does not have user SSH access. You or your network administrator should be able to set a static IP address for the robot based on its MAC address (via your department's DHCP server). The robot's MAC address is available on the [about page](https://iroboteducation.github.io/create3_docs/webserver/about/#ifconfig).

**Reply by shamlian** [reply]
What's the question behind the question? That is, why do you need a static IP? Are you on a multicast-blocking network and need to set up RMW configuration files for static IPs? Can you do it by robot hostname instead?
